onos-topo/Chart.yaml

```diff
@@ -3,8 +3,8 @@ name: onos-topo
 description: ONOS Topology service
 kubeVersion: ">=1.17.0"
 type: application
-version: 1.1.1
-appVersion: v0.8.0
+version: 1.1.2
```

Comment:
It looks like I never bumped this. Thomas, can we make this 1.1.100?

Reply:
@ray-milkey, @adibrastegarnia  has asked me to bump the version in another PR [here](https://github.com/onosproject/onos-helm-charts/pull/497). Could you review the PR?